source/playbooks/overview.rst

```diff
@@ -25,31 +25,45 @@ For participants, it prescribes processes such as task checklists, status update
 
 Playbooks also allow teams to incorporate learnings from the retrospective to tweak and improve the playbook with every iteration.
 
-What’s a playbook?
-^^^^^^^^^^^^^^^^^^
+What's a playbook?
+------------------
 
-A playbook is a checklist of the tasks that make up your processes. These tasks can range from at-mentioning a team member, to using a slash command to start a Zoom call. Tasks can be assigned to users, they can be given due dates.
+A playbook is a repeatable process that is refined over time. For example, the steps you follow when dealing with an outage, a software release, or welcoming a new member of your team can all be made into a playbook.
 
-There are other parts of a playbook, such as automation settings, and metrics. But the very first thing you’ll want to set up is a checklist.
+Playbook usage can be measured and adapted, and you can follow a playbook each time it's run.
 
-Each time you use the process you’ve documented, such as onboarding a new customer, the playbook is used to start a run - a discrete single use of the process - and that run is captured in a dedicated channel. 
+What's a run?
+-------------
 
-Setting up a playbook includes configuring how the playbook manages the creation of its channel as well as how stakeholders are notified.
-To open a playbook and view its statistics, select the playbook name. To begin a run using a specific playbook, select **Run** beside that playbook’s name.
+A run is the execution of the steps in a playbook. When a playbook is run, the process begins in a channel. Members of a playbook can view the run's progress in the run details page or they can participate in the run channel.
 
-Templates
-~~~~~~~~~
+When the process is completed, the run is ended. A retrospective can be run and the channel can be archived.
 
-Creating a playbook from scratch can be daunting, even if you have the process mapped out. One way to get started quickly is to use one of the pre-configured templates available. These templates are populated with content and settings to provide guidance and are customizable.
+Keywords
+--------
 
-Playbook templates are basic workflows that you can use to get started quickly. As you learn more about your workflows, you can customize them into specific playbooks.
+It's important to make it easy to start a run. One way to do this is by setting up keywords. These keywords prompt a user to start the run when they're used. In the incident response playbook, the keywords are specific to critical incidents, for example ``sev-1`` and ``#incident``. It's unlikely that someone would use those terms in general conversation and, even if they do, they can elect not to start the playbook run when prompted.
```

Comment:
AFAIK keywords are configured in a channel, where a user could be prompted to start a run from a particular playbook when a keyword is written.

keywords are not linked to the playbook or the run, but the channel.

I think you already know this but I understand from the previous text that some playbooks are configured with keywords.